backend/api/apis/failure/v1alpha1/failure.go

```diff
@@ -1,15 +1,23 @@
 package v1alpha1
 
-import "github.com/google/uuid"
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
 
 type Failure struct {
-	TeamName     string    `json:"team"`
-	TeamID       uuid.UUID `json:"team_id"`
-	JiraID       uuid.UUID `json:"jira_id"`
-	JiraKey      string    `json:"jira_key"`
-	JiraStatus   string    `json:"jira_status"`
-	ErrorMessage string    `json:"error_message"`
-	Frequency    float64   `json:"frequency"`
+	TeamName      string    `json:"team"`
```

Comment:
Separate every data with a space and comment

Reply:
not sure I got this comment , can you give an example ? 
type Failure struct {
```
	TeamName      string    `json:"team"`
	TeamID        uuid.UUID `json:"team_id"`
	JiraID        uuid.UUID `json:"jira_id"`
	JiraKey       string    `json:"jira_key"`
	JiraStatus    string    `json:"jira_status"`
	ErrorMessage  string    `json:"error_message"`
	Frequency     float64   `json:"frequency"`
	TitleFromJira string    `json:"title_from_jira"`
	CreatedDate   time.Time `json:"created_date"`
	ClosedDate    time.Time `json:"closed_date"`
	Labels        string    `json:"labels"`
```

Reply:
every key should be commented

Reply:
Fixed and added comment for each key